cx-sig-charter-draft.md

```diff
@@ -0,0 +1,81 @@
+# Contributor Experience Special Interest Group Charter
+
+This charter uses the Roles and Organization Management outlined in [sig-governance](https://github.com/spinnaker/governance/blob/master/governance.md).
+
+## Scope
+
+The Contributor Experience Special Interest Group (SIG) is responsible for improving the experience of those who contribute upstream to the Spinnaker project. We do this by creating, and maintaining programs and processes that promote community health and reduce project friction while retiring those programs and processes that don't. Being conscientious of our contributor base is critical to scaling the project, growing the ecosystem, and helping the project succeed.
+
+We do this by listening - whether it’s through our roadshows to SIG meetings, surveys, data, or [GitHub issues](https://github.com/spinnaker/spinnaker/issues), we take in the feedback and turn it into our project list. We build a welcoming and inclusive community of contributors by giving them places to be heard and productive.
+
+### In scope
+
+#### Projects & Services
+
+-  In accordance with the code of conduct, establish policies, standards, and procedures for the use, moderation, and management of all public platforms officially used by the project, including but not limited to:
+  - [GitHub Management](https://github.com/spinnaker)
+  - [Slack](https://join.spinnaker.io/)
+  - [Spinnaker](https://www.youtube.com/channel/UCcxQbw8kT1-FRhFhO2QCetg) YouTube channel
+  - Zoom
+  - Google Meet
+- Set up alerting automation for GitHub events, including new contributors, new issues, and new pull requests.
+- Work with other SIGs and interested parties in the project to execute GitHub tasks where required, see [GitHub Management](https://github.com/spinnaker) for more detail
+- Own and execute events that are targeted to the Spinnaker contributor community, including:
+  - Contributor Summit(s) 
+  - [Gardening Days and hackathons](https://github.com/spinnaker-hackathon/gardening)
+  - Retrospective moderation for other SIGs upon request
+  - Other events, like other SIG face to face events, upon request and consideration
+- Strategize, build, and execute on scalable mentoring programs for all contributor levels. These may include:
```

Comment:
FYI: https://github.com/spinnaker/governance/pull/51

Reply:
Yes #51 is a good issue. What this meant by mentorship here is 1:1 and group mentoring. The idea is a new user comes into the community and test the waters and when they are ready a mentor will guide them within a SIG. Again, very much like we do in Jenkins and Kubernetes.